src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -1014,6 +1014,7 @@ export default {
 	display: flex;
 	padding: 8px 4px 8px 12px;
 	align-items: center;
+	border-bottom: 1px solid transparent;
 
 	&--scrolled-down {
 		border-bottom: 1px solid var(--color-placeholder-dark);
```

Comment:
That also would be a nice addition to change specific parameter only
```suggestion
		border-bottom-color: var(--color-placeholder-dark);
```